distribution/docs/src/main/resources/content/_architectures/_eventing/eventing-intro.adoc

```diff
@@ -43,10 +46,4 @@ The key components of ${branding} Eventing include:
 * Delivery Method
```

Comment:
✏️ Maybe add links to `Subscription` and `Delivery Method` here now that these sections aren't right after this section

Reply:
Actually, this might not work because the `Subscription` and `Delivery Method` are not included in the architecture doc.

Reply:
I'll go ahead and link them anyway. There will be more future work to differentiate between the `architecture reference` section, which should explain how things work together, and the `developing` section, which instructs on actually developing new components